test/benchmarks/netperf/netperf-rc.yaml.in

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    app: {{ podname }}
+    
+  name: {{ podname }}
+  namespace: netperf
+spec:
+  replicas: 1
+  selector:
+    app: {{ podname }}
+  template:
+    metadata:
+      labels:
+        app: {{ podname }}
+    spec:
+      # Force this pod onto the specified node
+      nodeSelector:
```

Comment:
nit. I believe you can assign Node to the Pod directly, and skip the nodeSelector and scheduler altogether.